lib/tasks/tasks.rake

```diff
@@ -9,4 +9,9 @@ namespace :tasks do
       end
     end
   end
+
+  desc "Sync subjects"
+  task sync_subjects: :environment do
+    Notification.subjectable.without_subject.each{|n| n.send(:update_subject) }
```

Comment:
* Should probably do this [in batches](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_in_batches), as I have over 10,000 notifications just myself, so on octobox.io this task might not even work due to all the potential initializations.

* Why `send?`

* Just a thought... Given the number of notifications a person has, this could be pretty bad for someone's rate limit if they have so many notifications. This will be mitigated heavily by one person's notification syncing the subject for everyone else's notification, but 2 seconds might be too fast if the server has too many of them.

Maybe we should have `update_subject` take in a seconds parameter to tweak the distance in time determined to skip syncing a subject. Default it to 2 seconds as it's normally written, but in this rake task specify something quite a bit longer to maximize the potential to skip pointless calls.